frontend/src/components/VFeatureNotice/VDarkModeFeatureNotice.vue

```diff
@@ -11,9 +11,8 @@ const handleClick = () => {
 </script>
 
 <template>
-  <!-- TODO: Populate the post permalink. -->
   <VFeatureNotice
-    see-more-href="https://make.wordpress.org/openverse/"
+    see-more-href="https://make.wordpress.org/openverse/?p=2216"
```

Comment:
I see this as the permalink of said post.
```suggestion
    see-more-href="https://make.wordpress.org/openverse/2024/12/09/launching-the-dark-mode/↗"
```

Reply:
@krysal the link with `?p` goes to the post for me. The permalink you provided does not work for me. Presumably because it can change depending on when the post is actually published?

Reply:
I'm not certain, but I would expect the date to change after publishing. I think we can also change the link later, if needed.

Reply:
Right, it's better to leave the shortlink! Dismiss this.